README.md

```diff
@@ -0,0 +1,54 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Capstone Project on Behance Design
+
+> This is Microverse's first capstone project, where I built a mobile and desktop website for a fictional golf club event implementing the concepts I have earned in the first module: CSS, HTML, and JavaScript
+
+## Built With
+
+- HTML 5
+- CSS 3
+- JAVASCRIPT
+
+# Live Demo Link
+
+[Live Demo Link](https://abreham1222.github.io/capstone-project/)
+
+## Project Introduction
+
+[Presentation Loom Video](https://www.loom.com/share/67882df1be504321ab479ee61f2e4a4f)
+
+## Getting Started
+
+**Just clone the project by runing the `https://github.com/abreham1222/capstone-project.git` command in your command line, and you are done.**
+
+## Author
+
+👤 **Abreham Awoke**
+
+- GitHub: [@githubhandle](https://github.com/abreham1222)
+- Twitter: [@twitterhandle](https://twitter.com/Abreham1222)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/abreham1222)
+
+# Getting Started
+
+**This project is the presentation of a fictional event website for mobile and desktop. It was built based on the [original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) idea by Cindy Shin in Behance applying the acquired knowledge about JavaScript, HTML, and CSS through this module.**
```

Comment:
- [ ] Kindly add this under the acknowledgments section to make your readme organized.